### PR TITLE
adding track list menu

### LINKED ISSUE #182: Left side-nav to display playlist
Need a button on the top-left clicking on which will unfold a sidenav which will contain the tracks in sequence to be played, clicking on any particular track will play it.

- [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This issue is for enhancement/bug fixing.
- [ ] This issue is an addition to JSON database.
- [x] I have gone through [Contribution Guidelines](https://github.com/Anshuman-Verma/ongaku/blob/master/CONTRIBUTING.md).
- [ ] Assign this issue to me.

### PR DESCRIPTION
**Pull request by joseph-perez**
Fixes #182 #79 

- [x] This issue is for enhancement/bug fixing.
- [ ] This issue is an addition to JSON database.
	- [ ] Pull Request Title is of form `Added [Anime Name]`.
- [ ] This addition is already on ongaku.
	- [ ] This PR is an improvement to current addition.
- [x] I have checked my addition locally and it is working and won't break existing entries.
- [x] I have gone through [Contribution Guidelines](https://github.com/anshumanv/ongaku/blob/master/CONTRIBUTING.md).
- [x] This pull request will be closed if I failed to update it even once in a week.

### CODE REVIEW

**Comment by anshumanv** [comment]
Hey @joseph-perez thanks for contributing here, it works amazingly well, just the way I wanted.
One bug that persists is on selecting a song from the list the content above it should move at the end, please fix that. 😄

File: js/ongaku.js [hunk 1]
```diff
@@ -263,17 +280,41 @@ window.addEventListener("load", function () {
...
+			track_list += '<li>'+order[i].name+'</li>';
+		}
+
+		document.querySelector('#track-list ul').innerHTML = track_list;
+		$('#track-list ul li').on('click', select_track);
+	}
+
+	function select_track(e) {
+		//finding the selected title
+		var selected_title = e.target.innerHTML;
```
Comment (anshumanv) [inline js/ongaku.js]: can you please use let instead ?

Comment (joseph-perez) [reply js/ongaku.js]: Sure thing! I'm late to the game on all the es6 hotness haha 🔥

**Comment by joseph-perez** [comment]
Sorry about the bug 😅  I pushed the fixes you requested and I think it's looking good now 👍

**Comment by anshumanv** [comment]
Thanks for the quick fix @joseph-perez , it works amazingly well.
Thanks for the PR. 👍

**Review by anshumanv — APPROVED** [review]
👍

**Pull request merged**